web/interfaces.js

```diff
@@ -107,14 +107,20 @@ class IPDFLinkService {
  */
 class IPDFHistory {
   /**
-   * @param {Object} params
+   * @param {Object} options
+   * @param {*} options.fingerprint
+   * @param {boolean} options.resetHistory
+   * @param {boolean} options.updateUrl
    */
   initialize({ fingerprint, resetHistory = false, updateUrl = false }) {}
 
   reset() {}
 
   /**
    * @param {Object} params
+   * @param {*} params.namedDest
+   * @param {*} params.explicitDest
+   * @param {number} params.pageNumber
```

Comment:
Again, please see https://github.com/mozilla/pdf.js/blob/bf5a45ce6d0e9d070204859a39844bcdaf69e9aa/web/pdf_history.js#L44-L50 for the expected types.